### DISCUSSION TITLE
Display formatted data (xml/json)

### DISCUSSION THREAD

**Discussion by sbarilov** [twilio-labs/paste#2559] [General] [answered]
Any Paste component or any tips/guidance how we ca display formatted text as xml.
We are migrating some front end components and need to be able to do something like this:

![Screen Shot 2022-07-26 at 12 00 20 PM](https://user-images.githubusercontent.com/104868025/181090475-15040686-df76-4994-af92-cd0238cb5070.png)

I wasn't able to find any Paste component that could help.

**Comment by shleewhite** [comment, marked as answer]
Hi @sbarilov!

We are considering adding t[he Snippet component from Match](https://match.twilio.design/components/snippet/) to Paste. Would this component work for your use case?

**Reply by sbarilov** [reply]
Hi, thanks for getting back to me,
Yes, if that helps to display XML/HTML/JSON - this would be helpful.

**Reply by kate-porter** [reply]
Great. It will be able to support displaying those languages. We've actually got a ticket for this component in our current sprint, so we'll follow up with you once it's available!

**Reply by sbarilov** [reply]
oh, that sounds great!

**Reply by SiTaggart** [reply]
The code block component made it out to Paste core https://paste.twilio.design/components/code-block
